src/core/paths.jl

```diff
@@ -47,8 +47,8 @@ function _fix_comp_path!(child::AbstractComponentDef, parent::AbstractCompositeC
         # Fix internal param conns
         conns = child.internal_param_conns
         for (i, conn) in enumerate(conns)
-            src_path = ComponentPath(child_path, tail(conn.src_comp_path))
-            dst_path = ComponentPath(child_path, tail(conn.dst_comp_path))
+            src_path = ComponentPath(child_path, conn.src_comp_path.names[end])
+            dst_path = ComponentPath(child_path, conn.dst_comp_path.names[end])
```

Comment:
I suppose I also could have done this instead
```
 src_path = ComponentPath(head(child_path), conn.src_comp_path)
```

But since this works I guess it doesn't matter....